src/app/pages/CpsSty/categoryMap/index.test.js

```diff
@@ -0,0 +1,9 @@
+import categoryMap from './index';
+
+test('should assign the correct value', () => {
+  expect(categoryMap('Analysis')).toEqual('AnalysisNewsArticle');
+});
+
+test('should assign NewsArticle when category is null', () => {
+  expect(categoryMap('')).toEqual('NewsArticle');
+});
```

Comment:
Perhaps add a test for when the category is not recognised: `expect(categoryMap('random category')).toEqual('NewsArticle');`